plugin/grpc_provider.go

```diff
@@ -105,6 +105,13 @@ func (p *GRPCProvider) getDatasourceSchema(name string) providers.Schema {
 	return dataSchema
 }
 
+// getProviderMetaSchema is a helper to extract the schema for the meta info
+// defined for a provider,
+func (p *GRPCProvider) getProviderMetaSchema() providers.Schema {
```

Comment:
The client implements this in more methods than the server, but still seems to be missing `ReadResource`

Reply:
:+1: thanks!